homeassistant/components/vesync/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow utilities."""
+import logging
+from collections import OrderedDict
+import voluptuous as vol
+from homeassistant import config_entries
+from homeassistant.core import callback
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@callback
+def configured_instances(hass):
+    """Return already configured instances."""
+    return [
+        entry.data[CONF_USERNAME]
```

Comment:
We don't seem to check against username. We could just return a list of domain entries.